### PR TITLE
New Help section

### PR DESCRIPTION
**Pull request by briannailiev**
Section and content has been added to address FAQs

### CODE REVIEW

File: website/pages/en/help.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      <li>Merchant ID</li>\
+      <li>Product or service you provide</li>\
+  </ul>'
+},
+{
+  title: 'Can I integrate SumUp to my website?',
+  content: '<p>Yes. As a SumUp customer, you have the option to integrate SumUp’s eCommerce service with a webshop plugin or via direct integration.</p>\
+  <p>3rd party plugins supported by SumUp Partners are built with WooCommerce / Wordpress and PrestaShop.</p>\
+  <ul>\
+      <li>For <a href="https://codecanyon.net/item/woocommerce-sumup-payment-gateway/23109535">WooCommerce / Wordpress</a></li>\
```
Comment (lyubomir-sumup) [inline website/pages/en/help.js]: The following attributes have to be added to the link:
`target="_blank" rel="noopener"`

File: website/pages/en/help.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  </ul>\
+  <ul>\
+      <li>For <a href="https://addons.prestashop.com/en/other-payment-methods/44630-sumup-payment.html">PrestaShop</a></li>\
```
Comment (lyubomir-sumup) [inline website/pages/en/help.js]: Same as the above

File: website/pages/en/help.js (hunk 1 above)
Comment (briannailiev) [reply website/pages/en/help.js]: Done

File: website/pages/en/help.js (hunk 2 above)
Comment (briannailiev) [reply website/pages/en/help.js]: Done

**Pull request merged**